### PR TITLE
686 admin burger menu

### PR DESCRIPTION
**Pull request by DmytroStoliaruk**
dev
## Zerowaste

* [#686 ]

## Code reviewers

- [ ] @loqimean 

### Second Level Review

- [ ] @iryna-borniak 

## Summary of issue

Refactor admin menu from Bootstrap to Tailwind styles
Fix the unworking burger menu

## Summary of change

1. Refactored admin menu
2. Add burger menu, add Stimulus controller for it

https://github.com/ita-social-projects/ZeroWaste/assets/35365419/8a251cfc-5ecf-4b00-a12c-b87d5ebeaead

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by iryna-borniak — APPROVED** [review]

File: app/views/layouts/_account_navbar.html.erb [hunk 1]
```diff
@@ -1,91 +1,93 @@
...
-        <span class="toggler-icon top-bar"></span>
-        <span class="toggler-icon middle-bar"></span>
-        <span class="toggler-icon bottom-bar"></span>
-      </button>
-    </div>
+      <div>
+        <button data-controller="burger" data-action="click->burger#toggle" class="admin-menu-burger-button">
+          <svg class="block h-8 w-8" stroke="currentColor" stroke-width="2">
+            <path data-burger-target="icon"></path>
+          </svg>
```
[the remark this answers is not in the document]
Comment (DmytroStoliaruk) [reply app/views/layouts/_account_navbar.html.erb]: зробив

File: app/javascript/controllers/burger_controller.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["icon"];
+
+  burgerPath = "M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z";
+  crossPath = "M3 3L21 21M3 21L21 3";
```
[the remark this answers is not in the document]
Comment (DmytroStoliaruk) [reply app/javascript/controllers/burger_controller.js]: ок

File: app/javascript/controllers/burger_controller.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["iconLines", "iconCross"];
+
+  toggle() {
+    document.getElementById('navbarNav').classList.toggle('hidden');
```
Comment (loqimean) [inline app/javascript/controllers/burger_controller.js:7]: таргети)

**Review by loqimean — CHANGES_REQUESTED** [review]

File: app/javascript/controllers/burger_controller.js (hunk 3 above)
Comment (DmytroStoliaruk) [reply app/javascript/controllers/burger_controller.js]: виправив

**Review by loqimean — APPROVED** [review]

**Pull request merged**